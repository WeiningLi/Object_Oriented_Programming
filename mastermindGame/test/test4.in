KAUHLIUHW▒~@~B▒~@~B/
981209730912
ABDBBDBDBDF
AFFFA
B
AAAAAA
BBBBBB
DDDDDDDDDDDDD
FFFFFF
AFAFAFAF
FFFFFF
CCCCCC
N

