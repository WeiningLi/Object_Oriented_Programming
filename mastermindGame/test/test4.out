Welcome to Mastermind! Please enter your first guess:
1b,1w
6 guesses left. Enter guess:
0b,1w
5 guesses left. Enter guess:
0b,2w
4 guesses left. Enter guess:
0b,1w
3 guesses left. Enter guess:
2b,0w
2 guesses left. Enter guess:
1b,0w
1 guesses left. Enter guess:
1b,0w
You lost! The password was: AAECD
Would you like to play again? (Y/N): 
